lib/routes/gamer/gnn-index.ts

```diff
@@ -137,7 +138,7 @@ async function handler(ctx) {
                         component = _$('div.text-paragraph').html();
                     }
                 }
-                item.pubDate = timezone(parseDate(dateStr, 'YYYY-MM-DD HH:mm:ss'), +8);
+                item.pubDate = timezone(parseDate(dateStr, 'YYYY-MM-DD HH:mm:ss'), 0);
```

Comment:
I don't think the date from articles are from UTC +0.

Reply:
Sorry, I got confused about the time zone.